editors/code/src/config.ts

```diff
@@ -156,6 +156,7 @@ export class Config {
     get featureFlags() { return this.cfg.get("featureFlags") as Record<string, boolean>; }
     get additionalOutDirs() { return this.cfg.get("additionalOutDirs") as Record<string, string>; }
     get rustfmtArgs() { return this.cfg.get("rustfmtArgs") as string[]; }
+    get vscodeLldb() { return this.cfg.get("vscodeLldb") as boolean; }
```

Comment:
```suggestion
```
I suppose this user-facing configuration doesn't make much sense (and for the record: all of these should also be declared in `package.json -> contributes -> configuration -> properties`).
I was just suggesting to encapsulate `vscode.extensions.getExtension("vadimcn.vscode-lldb") != null` expression itself into a getter on `Config`, though it may not be worth for this level of indirection.